apps/chat/src/components/Files/FileManagerModal.tsx

```diff
@@ -39,15 +39,17 @@ interface Props {
   initialSelectedFilesIds?: string[];
   allowedTypes?: string[];
   maximumAttachmentsAmount?: number;
-  isInConversation?: boolean;
+  customHeaderLabel?: string;
```

Comment:
let's make this required as discussed
```suggestion
  headerLabel: string;
```

Reply:
Changed as suggested.
Changes applied in all needed places.